app/styles/login.module.css

```diff
@@ -0,0 +1,85 @@
+.login__container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: white;
+  z-index: 9999;
```

Comment:
What is this 9999? won't maintaining become hard if random values are used?

Reply:
that part is there to hide and  header & footer and show login component instead.
i've used high value so it wont break in case other some other comp use z-index

Reply:
@Mir-SA I am not aligned with using some random values for the z index here.
Please come up with a better way here.

Reply:
got it. i'll check out existing z-index values on components in route and set this z-index slightly higher, if that's fine?
i had tried few other ways but they didnt work out, so went with this one.